README.md

```diff
@@ -204,7 +204,7 @@ Free Podcasts and Screencasts:
 
 ### Programming Playgrounds
 
-Write, compile and run your code within a browser. Try it out!
+Write, compile, and run your code within a browser. Try it out!
```

Comment:
I would revert that. That makes it worse

Reply:
Ok